deployments/terraform/modules/consumer/alarms.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_cloudwatch_metric_alarm" "success-rate" {
```

Comment:
the module names are quite generic right know.  
we also have to consider, that there are different consumers than sqs.  
i would suggest "grouping" the alarms with an "alarm-" prefix and to include input type.  
eg: `alarm_consumer_sqs`